packages/amplify-graphql-types-generator/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Remove this file. We have a License file on our mono repo whihc is Apcha Software and not MIT.

Reply:
Are these 2 compatible with one another?

Reply:
Removed